tutorials/2_sprite_animations_gestures/code/lib/main.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flame/gestures.dart';
+import 'package:flame/sprite.dart';
+import 'package:flutter/material.dart';
+import 'package:flame/game.dart';
+
+void main() {
+  final myGame = MyGame();
+  runApp(
+    GameWidget(
+      game: myGame,
+    ),
+  );
+}
+
+class MyGame extends Game with TapDetector {
+  late SpriteAnimation runningRobot;
+  late Sprite pressedButton;
+  late Sprite unpressedButton;
+
+  bool isPressed = false;
+
+  final buttonPosition = Vector2(200, 120);
+  final buttonSize = Vector2(120, 30);
+
+  final runningPosition = Vector2(240, 50);
+  final runningSize = Vector2(48, 60);
+
+  @override
+  Future<void> onLoad() async {
+    runningRobot = await loadSpriteAnimation(
+      'running.png',
+      SpriteAnimationData.sequenced(
```

Comment:
unrelated but I wanted to rename this to have horizontal on the name, your tutorial reminded of that

Reply:
SGTM